server/datastore/mysql/software_test.go

```diff
@@ -964,19 +1125,19 @@ func testDeleteVulnerabilitiesByCPECVE(t *testing.T, ds *Datastore) {
 func testHostsByCVE(t *testing.T, ds *Datastore) {
 	ctx := context.Background()
 
-	hosts, err := ds.HostsByCVE(ctx, "CVE-2022-1234")
+	hosts, err := ds.HostsByCVE(ctx, "CVE-0000-0000")
 	require.NoError(t, err)
 	require.Len(t, hosts, 0)
 
 	insertVulnSoftwareForTest(t, ds)
 
 	// CVE of foo chrome 0.0.3, both hosts have it
-	hosts, err = ds.HostsByCVE(ctx, "cve-123-456-789")
+	hosts, err = ds.HostsByCVE(ctx, "CVE-2022-0001")
 	require.NoError(t, err)
 	require.Len(t, hosts, 2)
 
 	// CVE of bar.rpm 0.0.3, only host 2 has it
-	hosts, err = ds.HostsByCVE(ctx, "cve-321-432-543")
+	hosts, err = ds.HostsByCVE(ctx, "CVE-2022-0002")
```

Comment:
Thanks for taking the time to ("visually") sanitize the tests :)

Reply:
When I was debugging failing tests, I was having trouble visually comparing the cve's (mild dyslexia). I changed them to match the actual format for cve's and to make it easier to compare.